addons/html_editor/static/src/others/dynamic_placeholder/dynamic_placeholder.js

```diff
@@ -92,7 +92,12 @@ export class DynamicPlaceholderPlugin extends Plugin {
             return;
         }
 
-        let dynamicPlaceholder = "object." + chain;
+        let dynamicPlaceholder;
+        if (chain.includes("date")) {
+            dynamicPlaceholder = `format_datetime(object.${chain}, tz=object.partner_id.tz)`;
+        } else {
+            dynamicPlaceholder = "object." + chain;
+        }
```

Comment:
can we use ternary over here?

Reply:
Hello @thsh-odoo 

Changes are done.
Please let me know if I have missed something.

Thanks :smile: